dev-support/docker/pkg-resolver/packages.json

```diff
@@ -263,6 +352,14 @@
       "openjdk-11-jdk",
       "openjdk-17-jdk"
     ],
+    "debian:12": [
+      "temurin-17-jdk",
+      "temurin-24-jdk"
+    ],
+    "debian:13": [
+      "temurin-17-jdk",
+      "temurin-24-jdk"
```

Comment:
temurin-25 is out

BTW, I think we should prefer to use the JDK provided by official APT repo if possible, Debian 13 already has `openjdk-25-jdk`